felix/bpf/ipsets/ipsets.go

```diff
@@ -370,9 +380,11 @@ func (m *bpfIPSets) markIPSetDirty(data *bpfIPSet) {
 	m.dirtyIPSetIDs.Add(data.ID)
 }
 
-func (m *bpfIPSets) SetFilter(ipSetNames set.Set[string]) {
-	// Not needed for this IP set dataplane.  All known IP sets
-	// are written into the corresponding BPF map.
+func (m *bpfIPSets) SetFilter(fn func(ipSetName string) bool) {
+	m.filterIPSet = fn
+}
+
+func (m *bpfIPSets) MarkDirty(ipsetNames set.Set[string]) {
```

Comment:
Feel a bit uneasy that we have a flexible filter function but we're not implementing the rescan here.  Someone could come along later and use the filter function without realising this isn't implemented.

Should at least comment "Not implemented because the only filter we use with this IP set is 100% static, based on the IP set name."